src/main/java/nl/workingtalent/wtacademy/user/User.java

```diff
@@ -42,7 +36,7 @@ public class User {
 	@OneToMany(mappedBy = "user")
 	private List<Reservation> reservations;
 	
-	@OneToMany(mappedBy = "user")
+	@OneToMany(mappedBy = "user", fetch = FetchType.EAGER)
```

Comment:
Waarom deze verandering? Wat doet FetchType eager?

Reply:
Zonder dit kun je niet zomaar de geleende boeken van een user opvragen in je code. Is gebruikt om te checken of een gebruiker een boek heeft gelezen en heeft terug gebracht